options.go

```diff
@@ -97,6 +99,18 @@ func CookieName(name string) Option {
 	}
 }
 
+// TrustedOrigins configures a set of origins (referrer) that are trusted for
```

Comment:
s/referer/referers (pluralize; single 'r' as referer is an accidental mispelling)

I would also lead with the use case - e.g.

```
// TrustedOrigins configures a set of origins (Referers) that are considered as trusted.
// This will allow cross-domain CSRF use-cases - e.g. where the front-end is served 
// from a different domain than the API server - to correctly pass a CSRF check.

// You should only provide origins you own or have full control over.
```

Reply:
Also - add a small section to the README under the https://github.com/gorilla/csrf#javascript-applications heading that describes a scenario where this is useful. A couple of bullet points is OK.